components/Button/Button.module.scss

```diff
@@ -1,11 +1,15 @@
 .button {
-  color: #ff9900;
-  font-weight: 600;
```

Comment:
se a gente quiser font-weight diferente, é só alterarmos na classe do {children} que for enviado